docs/docs/setting-up-gatsby-without-gatsby-new.md

```diff
@@ -0,0 +1,66 @@
+## Setting Up a Gatsby Site Without Gatsby New
+
+There are many Enterprise level companies that maintain an internal clone of the NPM registry for security purposes. If you work for such a company, you may find that you are able to successfully run `npm install -g gatsby-cli` but cannot run the `gatsby new <project-source>` as the `gatsby new` command clones a repo from a public GitHub repository. Many companies block public GitHub, which will cause the `gatsby new` command to fail. Not to worry, though, you can set up a new Gatsby site without the `gatsby new` command with a few quick steps.
+
+## Preparing your environment
+To get started with Gatsby, you’ll need to make sure you have the following software tools installed:
+
+1. [Node.js](https://www.gatsbyjs.org/tutorial/part-zero/#install-nodejs)
+1. [npm CLI](https://www.gatsbyjs.org/tutorial/part-zero/#familiarize-with-npm)
+1. [Gatsby CLI](https://www.gatsbyjs.org/tutorial/part-zero/#install-the-gatsby-cli)
+
+For step-by-step installation instructions and detailed explanations of the required software, head on over to the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-zero/).
+
+After your developer environment is set up, you'll want to set up a new project folder.
+```shell
+mkdir my-new-gatsby-site
+cd my-new-gatsby-site
+```
+
+Next, you'll need to set up NPM within your project.
+```shell
+npm init
+```
+
+Fill out the prompts for the package.json file that is generated. If you'd like to skip that, you can run `npm init -y` and a pre-filled package.json will be generated for you.
+
+Now, you'll need to install the neccessary packages that Gatsby relies on to work its magic.
+```shell
+npm i gatsby react react-dom
+```
+
+Once your packages have installed, you'll likely want to add some of the Gatsby specific files at the root of your project.
```

Comment:
I think we can remove this part as these files are not necessary to get a Gatsby site working